src/main/java/core/basesyntax/repoter/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.repoter;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String HEADER = "fruit,quantity";
+    private static final String DEFAULT_DELIMITER = ",";
+    private StringBuilder builder;
```

Comment:
The field 'builder' should be a local variable inside the 'getReport' method to avoid shared state between method calls which can lead to potential concurrency issues.

Reply:
not fixed